src/components/Loginform/Loginform.jsx

```diff
@@ -0,0 +1,16 @@
+import './LoginForm.scss'
+
+function LoginForm() {
+  return (
+  <div className="LoginForm">
+      <h2>Pierakstīšanās forma</h2>
+      <label for="username"><b>Lietotājvārds</b></label>
+      <input type="text" placeholder="Enter Username" id="username" required />
+      <label for="password"><b>Parole</b></label>
+      <input type="password" placeholder="Enter Password" id="password" required />
+      <button onclick="login()">Pierakstīties</button>
```

Comment:
onclick event name is wrong, it should be `onClick`  - it is important
login() function itself should be before `return` statement